server/utils/utils.test.ts

```diff
@@ -472,3 +473,10 @@ describe('isFutureDate', () => {
     expect(result).toEqual(false)
   })
 })
+
+describe('calculateAge', () => {
+  test('calculate age from date of birth', () => {
+    const twentyYearsAgo = moment().subtract(20, 'years').format('YYYY-MM-DD')
+    expect(calculateAge(twentyYearsAgo)).toBe(20)
+  })
```

Comment:
Nice to have some boundary tests here - whats slightly less than 20 years ago, slightly more?